src/libraries/Microsoft.Extensions.Configuration/src/ConfigurationManager.cs

```diff
@@ -43,17 +43,13 @@ public ConfigurationManager()
         {
             get
             {
-                lock (_providerLock)
-                {
-                    return ConfigurationRoot.GetConfiguration(_providers, key);
-                }
+                using var reference = _providerManager.GetReference();
```

Comment:
(nit) `var` shouldn't be used here since the type isn't obvious from the left hand side.